drizzlepac/photeq.py

```diff
@@ -591,9 +586,6 @@ def photeq(files='*_flt.fits', sciext='SCI', errext='ERR',
             _log.removeHandler(h)
 
 
-#--------------------------
-# TEAL Interface functions
```

Comment:
Same as previous comment in a different module.  I would leave this for now with an added comment about TEAL being disabled and removing this text and/or functionality if the future (or something like this).